clvm_tools/curry.py

```diff
@@ -14,10 +14,8 @@
 
 CURRY_OBJ_CODE = assemble(
     """
-    ((c (q ((c 4 (c 2 (c 5 (c 7 (q ()))))))) (c (q ((c (c (q #c) (c (c (q #q) (c 5 (q ())))
-    (c ((c 6 (c 2 (c 11 (q (1)))))) (q ())))) (q ())) (c (i 5 (q (c (q #c) (c (c (q #q)
```

Comment:
@richardkiss, What is the significance of the of the `#` in `#q` and `#c` ?

Reply:
They used to be required – they no longer are. It emphasizes that this isn't a literal, but rather the byte string that corresponds to the `c` or `q` operator.